### PR TITLE
Clean drafts before each GMail test

### LINKED ISSUE #3315: accumulating drafts cause live gmail tests to fail
Example failure: https://flowcrypt.semaphoreci.com/jobs/3a01ea85-c237-4140-b189-34108f5d5b08

The solution could be to call an API that deletes all drafts on Gmail before start of the test. However, the test code that runs on nodejs does not have the right permission, only the browser extension does (after the login step). There are three options how to solve it that come to mind:

a) somehow extract the necessary access token from the browser extension after login step, and then make a call from NodeJS to delete all drafts.
b) create a separate miniapp for tests where the nodejs app will use puppeteer to log in and authenticate. The resulting code will be scraped from the window after authentication, then nodejs requests access token manually, and then uses it to delete the drafts.
c) find out why are we leaving drafts behind in some other test, and adjust the test to always make sure to delete the draft using Gmail UI at the end of a test

image

### PR DESCRIPTION
**Pull request by rrrooommmaaa**
This PR extracts google token and deletes all drafts on the account

close #3315

### CODE REVIEW

**Comment by rrrooommmaaa** [comment]
> a) somehow extract the necessary access token from the browser extension after login step, and then make a call from NodeJS to delete all drafts.

@tomholub I suggest to extract google token like this and clear the drafts using it (if that's what you meant)

File: test/source/browser/controllable.ts [hunk 1]
```diff
@@ -624,6 +624,18 @@ export class ControllablePage extends ControllableBase {
       }
     }
   }
+
+  public getFromLocalStorage = async (keys: string[]): Promise<any> => {
```
Comment (tomholub) [inline test/source/browser/controllable.ts:628]: Nice!! I did not know this was possible. This is an amazingly elegant solution.

Comment (rrrooommmaaa) [reply test/source/browser/controllable.ts:628]: Can we use https://www.npmjs.com/package/googleapis package to access gmail api?

Comment (tomholub) [reply test/source/browser/controllable.ts:628]: Sure, I'm pretty liberal with adding deps to the test infrastructure.

**Comment by rrrooommmaaa** [comment]
This version does requests cleanup on each `TestWithBrowser` call with `!isMock`

**Review by tomholub — APPROVED** [review]
Looks good

**Pull request merged**